docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,106 @@
+
+[![QuickStart test]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml)&nbsp;&nbsp;&nbsp;
+
+
+!!! tip "Estimated time to complete this example:" 
+    ~4 minutes (after installing prerequisites)
+
+## How to deploy and use KubeStellar
+
+This guide is intended to show how to (1) quickly bring up a **KubeStellar** environment using helm, (2) install plugins with brew, (3) retrieve your kubeconfig, (4) install KubeStellar Syncers on 2 edge clusters, (5) and deploy an example workload to both edge clusters.
+
+## 0. Pre-reqs
+
+- helm
+- brew
+- kubectl
+- kind clusters (3) 
+   - 1 KubeStellar core cluster (kind-core)
+   - 2 KubeStellar edge clusters (kind-edge-cluster1, kind-edge-cluster2)
+   
+## 1. Install KubeStellar
+
+```
+kubectl cluster-info --context kind-core
+helm install kubestellar --set EXTERNAL_HOSTNAME="localhost" --set EXTERNAL_PORT=1024
```

Comment:
The choice of "localhost" leads to a world of hurt and confusion when someone tries to use the kubeconfig fetched in step 3 from inside a container or VM or from another machine.  The NOTE below could be clearer. The point is not whether the user is "using DNS" in their environment. Everybody is "using" DNS in their environment. What needs to be said here is words about _must_ use a FQDN if the kubeconfig will be used from within a container or VM or other machine, how the FQDN must resolve to an IP address that the client can use to open a TCP connection that will reach the host of the hosting cluster, and yes, how this requires control over DNS records or /etc/hosts as relevant.